.github/workflows/build_and_test_workflow.yml

```diff
@@ -186,18 +349,18 @@ jobs:
         working-directory: ./artifacts
     strategy:
       matrix:
-        version: [ osd-2.0.0, osd-2.1.0, osd-2.2.0, osd-2.3.0, osd-2.4.0 ]
+        version: [ osd-2.0.0, osd-2.1.0, osd-2.2.0, osd-2.3.0 ]
```

Comment:
by accident?

Reply:
yep; bad cherrypick. will fix in a moment.